src/clj/game/cards-assets.clj

```diff
@@ -297,6 +297,19 @@
                                          (shuffle! state side :deck))}
                            card nil))}]}
 
+   "Kala Ghoda Real TV"
+   {:events
+    {:corp-turn-begins
+     {:optional
+      {:prompt "Use Kala Ghoda Real TV to look at the top card of the Runner's Stack?"
```

Comment:
Two of these bad boys installed trigger a duplicate prompt that doesn't behave too pretty. Since you can't use multiples of KGRT to look at the top, trash it and look again (trashing is in a paid ability window, different from turn start), I think adding some kind of uniqueness to this prompt would help.